python/libmdbx/mdbx.py.in

```diff
@@ -967,17 +968,17 @@ class MDBXError(enum.IntFlag):
     # The last of MDBX-added error codes
     MDBX_LAST_ADDED_ERRCODE = MDBX_TXN_OVERLAPPING
 
-    MDBX_ENODATA = 61
-    MDBX_EINVAL = 22
-    MDBX_EACCESS = 13
-    MDBX_ENOMEM = 12
-    MDBX_EROFS = 30
-    MDBX_ENOSYS = 38
-    MDBX_EIO = 5
-    MDBX_EPERM = 1
-    MDBX_EINTR = 4
-    MDBX_ENOFILE = 2
-    MDBX_EREMOTE = 15
+    MDBX_ENODATA = errno.ENODATA
```

Comment:
Unfortunately, these are not done for Windows, i.e. need a some `if` like such:
```python
    MDBX_ENODATA = if Windows then errno.ENODATA else errno.ERROR_HANDLE_EOF.
```

Reply:
Python does not have any NT error codes. They are all translated to posix error numbers internally to Python.

Reply:
So the only solution would be to again hard code those NT error codes. Would that be okay for you?

Reply:
I apologize for the delay with the answers. 

If, as you say, python virtualizes / translates system error codes, then it is necessary and sufficient to use this mechanism.
However, I don't see that there is a such conversion in a code.
E.g. a user wants to handle an error (catch an exception) in case there is no file.
Now it can't do this, because there is one common exception class, and the value of the error code may be different on Windows and non-Windows platforms.

So, I think there should be a code with this logic (in a pseudo code):
```
static function RaiseExceptionByErrorCode(ErrCode : integer)
  mdbx_err_msg = mdbx_liberr2str(ErrCode)
  if  mdbx_err_msg 
     raise MDBXErrorException(ErrCode, mdbx_err_msg)
 if is_on_windows
    raise OSError(winerr = ErrCode)
 else     
    raise OSError(errno = ErrCode)
```